easybuild/easyconfigs/r/RFdiffusion/RFdiffusion-1.1.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -1,57 +1,96 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/03
 easyblock = 'PythonBundle'
 
 name = 'RFdiffusion'
 version = '1.1.0'
 versionsuffix = '-CUDA-%(cudaver)s'
 
 homepage = 'https://github.com/RosettaCommons/RFdiffusion'
-description = """
-RFdiffusion is an open source method for structure generation, with or without conditional information
-(a motif, target etc). It can perform a whole range of protein design challenges as we have outlined
-in the RFdiffusion paper.
+description = """RFdiffusion is an open source method for structure generation, with or without
+conditional information (a motif, target etc). It can perform a whole range of
+protein design challenges as we have outlined in the RFdiffusion paper
+https://www.biorxiv.org/content/10.1101/2022.12.09.519842v1
 """
 
 toolchain = {'name': 'foss', 'version': '2022a'}
 
+_pysp = '%(installdir)s/lib/python%(pyshortver)s/site-packages'
+
 dependencies = [
-    ('CUDA', '11.7.0', '', SYSTEM),
     ('Python', '3.10.4'),
-    ('SciPy-bundle', '2022.05'),
-    ('e3nn', '0.3.3', versionsuffix),
     ('wandb', '0.13.4'),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch-bundle', '1.13.1', versionsuffix),  # same as used in DGL
```

Comment:
The tests are failing because the _main_ version of PyTorch in 2022a is `1.12.0`. For instance, the non-CUDA version of RFdiffusion (`RFdiffusion-1.1.0-foss-2022a.eb`) already depends on v1.12.0.
To avoid conflicts, this easyconfig should also use v1.12.0, or add a versionsuffix in the form of `PyTorch-1.13.1`.